airbyte-webapp/src/pages/ConnectionPage/pages/CreationFormPage/CreationFormPage.tsx

```diff
@@ -217,34 +217,33 @@ const CreationFormPage: React.FC = () => {
   )[type];
 
   return (
-    <MainPageWithScroll
-      headTitle={<HeadTitle titles={[{ id: titleId }]} />}
-      pageTitle={
-        <PageTitle
-          withLine
-          title={<FormattedMessage id={titleId} />}
-          middleComponent={<StepsMenu lightMode data={steps} activeStep={currentStep} />}
-        />
-      }
-    >
-      <FormPageContent big={currentStep === StepsTypes.CREATE_CONNECTION}>
-        {currentStep !== StepsTypes.CREATE_CONNECTION && (!!source || !!destination) && (
-          <ConnectionBlock
-            itemFrom={source ? { name: source.name, icon: sourceDefinition?.icon } : undefined}
-            itemTo={
-              destination
-                ? {
-                    name: destination.name,
-                    icon: destinationDefinition?.icon,
-                  }
-                : undefined
-            }
+    <>
+      <HeadTitle titles={[{ id: "sources.newSourceTitle" }]} />
+      <ConnectorDocumentationWrapper>
+        <>
```

Comment:
Fragment not needed

Reply:
I had set up the `ConnectorDocumentationPanel` to only accept a single ReactNode as a child.  That makes this fragment necessary.  If that seems unnecessary I can remove it and then remove this fragment.